### PR TITLE
Fix more than one value && the null scale; fixes #652

### LINKED ISSUE #652: IME exception in grapher while selecting item
### Prerequisites

- [X] I have written a descriptive issue title
- [X] I have verified that I am running the latest version of the CDP4-IME
- [X] I have searched [open](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description

Testing the grapher on a testing model (Types Testing on DEV server), created a top element to have an option tree, then opened the grapher, and selected items on the diagram (randomly)
After 3 or 4 clicks (selecting other items), there was an exception.

no logs available, only exception report.

### Steps to Reproduce

tbd if reproducible, by opening model with option trees, opening grapher, and randomly selecting items on the diagram consecutively

### Logs

  - [ ] There are no logs regarding the issue
  - [X] I have attached a log file or included the relevant section in this issue  

[IME exception on grapher - selecting item.txt](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/files/5660810/IME.exception.on.grapher.-.selecting.item.txt)
  

### System Configuration

- CDP4 version:
  - [X] CDP4-IME: 7.0.0 RC 5
  - [ ] Other: 
  -
- Web Services:
  - [ ] CDP4 Web Services version: 
  - [ ] OCDT WSP version: 
- Environment (Operating system, version and so on):
  - [ ] Microsoft Excel (32/64 bit): 
  - [ ] Windows OS (32/64 bit): 
  - [ ] Additional information:

### PR DESCRIPTION
**Pull request by nathanatrhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the CDP4-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/CDP4-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
 fixes #652 

- [x] Fixed the "More than one element" returned by SingleOrDefault on a Parameter or ParameterOverride ValueSet.
- [x] Fixed a NullReferenceException when the Parameter Scale is null
- [x] ElementParameterRowControlViewModel for display in the Parameter value overview of the grapher displays the actual value

### CODE REVIEW

**Review by samatrhea — APPROVED** [review]

**Review by alexatrhea — APPROVED** [review]

File: CDP4Composition/ViewModels/ParameterRowControlViewModel.cs [hunk 1]
```diff
@@ -181,7 +181,13 @@ private void UpdateProperties()
 
             this.Name = this.Parameter.ParameterType.Name;
             this.ShortName = this.Parameter.ParameterType.ShortName;
-            this.Value = $"{valueSet?.Published.FirstOrDefault()} [{this.Parameter.Scale.ShortName}]";
+            var actualvalue = valueSet?.ActualValue.FirstOrDefault() ?? "-";
```
Comment (alexatrhea) [inline CDP4Composition/ViewModels/ParameterRowControlViewModel.cs:184]: Why was this changed from published to actual?

Comment (nathanatrhea) [reply CDP4Composition/ViewModels/ParameterRowControlViewModel.cs:184]: I thought I made a mistake by displaying the published Value, When I had a look at the row view model from the ElementDefinition tree, I thought the Actual Value need to be shown, I'll roll back then

Comment (alexatrhea) [reply CDP4Composition/ViewModels/ParameterRowControlViewModel.cs:184]: roll back and leave Published. In theory the row control should probably show both and make a distinction between the two as "Value" can be vague.

**Review by alexatrhea — APPROVED** [review]

**Comment by alexatrhea** [comment]
@lxatrhea @nathanatrhea this can be merged into development right?

**Comment by lxatrhea** [comment]
> @lxatrhea @nathanatrhea this can be merged into development right?

I think so.

**Pull request merged**